### DISCUSSION TITLE
sh /vercel/path0/node_modules/.bin/tsc: Permission denied

### DISCUSSION THREAD

**Discussion by HugoAlbertBonet** [vercel/community#2051] [Help]
### Summary

I try to build my deployment from my Github repository (whose code works perfectly in my local host) and I receive the following output:
![image](https://user-images.githubusercontent.com/125313451/231451115-3de9575d-bf6d-4763-a6f4-8a00c26f77ba.png)

What can I do? I assume I have to grant execution permission to the server, how can I do that?

**Comment by swarnava** [comment]
Hi,

Could you share a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) repository that can be shared publicly as a git repository so that we can take a look?

**Reply by HugoAlbertBonet** [reply]
I can share with you the whole repository. Here you are: https://github.com/HugoAlbertBonet/wedding
